api/siad/server.go

```diff
@@ -69,6 +70,9 @@ func (s *server) consensusStateHandler(w http.ResponseWriter, req *http.Request,
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
+	// TODO: add marshalling to accumulators
+	cs.Elements = merkle.ElementAccumulator{}
+	cs.History = merkle.HistoryAccumulator{}
```

Comment:
These are missing, I was going to add an API response struct that omits these, but we are calling `InputSigHash` on the `siac` side.

Reply:
not sure I understand -- what exactly is missing? Both of these types have `MarshalJSON`/`UnmarshalJSON` methods.